dev_sandbox/main_windows.cpp

```diff
@@ -0,0 +1,112 @@
+#include "std/target_os.hpp"
+#if !defined(OMIM_OS_WINDOWS)
+#error Unsupported OS
+#endif
+
+#define GLFW_INCLUDE_NONE
+#include <GLFW/glfw3.h>
+#define GLFW_EXPOSE_NATIVE_WIN32
+#include <GLFW/glfw3native.h>
+
+#include <vulkan_wrapper.h>
```

Comment:
If vulkan_wrapper is included here directly, then it should be added in CMakeLists.txt as a direct dependency for this binary. The same goes for drape below.